src/sparsezoo/utils/onnx.py

```diff
@@ -102,7 +102,8 @@ def save_onnx(
          -  `external_data_file` is specified,
         then the external data of the model will be saved to `external_data_file`
         instead of `large_model_external_data_name`.
-    :param external_data_file: The optional name save the external data to.
+    :param external_data_file: The optional name save the external data to. Must be
```

Comment:
```suggestion
    :param external_data_filename: The optional name save the external data to. Must be
```
Maybe to be even more explicit. yes, the 'filename' argument in the onnx.save(...) is relative to the model location.